README.md

```diff
@@ -52,6 +52,13 @@ The following command injects the debugger into a process with a given PID that
 -m debugpy --listen localhost:5678 --pid 12345
 ```
 
+### Ignoring subprocesses
+The following command will ignore subprocesses started by the debugged process.
+```console
+-m debugpy --listen localhost:5678 -pid 12345 --config-subProcess False
```

Comment:
Another thing here that I didn't spot before: `-pid` should be `--pid`.

Reply:
Whoops. Fixed this.